Misc/NEWS.d/next/Library/2020-09-23-22-52-24.bpo-41842.lIuhC9.rst

```diff
@@ -0,0 +1,2 @@
+Add :func:`codecs.unregister` and :c:func:`PyCodec_Unregister` to unregister
+a codec search function.
```

Comment:
Hum, it would be more explicit if you add "function":

```suggestion
Add :func:`codecs.unregister` function to unregister
a codec search function.
```

Can you please add a second NEWS entry in the C API, for the PyCodec_Unregister function? Copy/paste the text and replace codecs.unregister with PyCodec_Unregister.

Reply:
make sense, updated.